### PR TITLE
Fix xtools template

### PR DESCRIPTION
**Pull request by crocket**
vimdiff from vim is required by xdiff

### CODE REVIEW

File: srcpkgs/xtools/template [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Template file for 'xtools'
 pkgname=xtools
 version=0.51
-revision=1
+revision=2
 noarch=yes
 build_style=gnu-makefile
-depends="bash curl findutils git make xbps"
+depends="bash curl findutils git make xbps vim"
```
Comment (Vaelatern) [inline srcpkgs/xtools/template]: And what about other flavors of vim?

Comment (crocket) [reply srcpkgs/xtools/template]: neovim doesn't come with vimdiff, but neovim can do diff with a different command.
Supporting other flavors of vim is out of scope for this pull request.

Comment (crocket) [reply srcpkgs/xtools/template]: I investigate further and discovered that `vim` is actually a symlink to either nvim or vim.
I don't know how to deal with vim flavors in `depends`, yet.

Perhaps, `virtualpkg` in `xbps.d` is the way to handle vim flavors?

**Comment by Duncaen** [comment]
I don't think its really necessary to have the hard dependencies, some user might not use `xdiff` at all and don't want to have vim.
Its easy enough to notice that vim is missing if you use `xdiff`.

**Comment by crocket** [comment]
What about optional dependency?

**Comment by Duncaen** [comment]
There are no optional dependencies.

**Comment by Duncaen** [comment]
None, its really not hard to use `find` or `ls` or the output of `xbps-install` and your preferred editor.
`xdiff` is just a small helper for people who can and want to use `vimdiff`.

**Comment by crocket** [comment]
Ok, I think I can make vimdiff or vim an optional dependency in xdiff.

**Pull request closed without merging**